src/lib.rs

```diff
@@ -22,9 +22,9 @@ pub use askar_storage as storage;
 pub use askar_storage::future;
 
 #[cfg(feature = "ffi")]
-mod ffi;
+pub mod ffi;
```

Comment:
I suggested it might be better to copy the tags module to the uniffi crate, to avoid exposing the ffi module here. It's some duplication, but if this ffi interface is deprecated then it wouldn't matter.

Reply:
Done

Reply:
I don't think this needs to be public in that case